inputs/wysiwyg.vue

```diff
@@ -472,6 +503,42 @@
         }
       }
 
+      /**
+       * Add hints via formatting to a length of text and more later
+       * @param {object} editor a Quill object
+       * @param {object} fmt a Quill format object
+       * @return {object}
+       */
+      function hint(editor, fmt = {}) {
+        let start,
+          end;
+
+        if (!editor.formatText || !editor.removeFormat) {
+          throw new Error('not a Quill instance');
```

Comment:
when would this error be thrown? (and what would be the steps to fix it?)

Reply:
I'll `console.error` this.